torchgeo/datasets/cowc.py

```diff
@@ -225,6 +265,40 @@ class COWCCounting(COWC):
     ]
     filename = "COWC_{}_list_64_class.txt"
 
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`VisionClassificationDataset.__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        .. versionadded:: 0.2
+        """
+        label_title = cast(str, sample["label"].item())
+
+        showing_predictions = "prediction" in sample
```

Comment:
Do we need to make this a variable if it's only being used once on the line below?

Reply:
No, but do we need to change it?

Reply:
Nope, I'm indifferent but it should at least be pointed out that it's not consistent with other plot methods.

Reply:
Other plots use this idea https://github.com/microsoft/torchgeo/blob/cbebc1e0dbb77fe37ce4bc8116792828fab8bc27/torchgeo/datasets/eurosat.py#L211